### PR TITLE
B 18795 int

### PR DESCRIPTION
**Pull request by deandreJones**
## Summary

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#zero-downtime-migrations).
- [ ] Have been communicated to #g-database.
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#secure-migrations).

### CODE REVIEW

File: migrations/app/schema/20240322144246_updateAddressesCounty.up.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+update
+	addresses adr
+set
+	county = (
+	select
+		max(usprc_county_nm) -- using max since some county names/zip combination have multiple records
+	from
+		us_post_region_cities usprc
+	where
+		adr.postal_code = usprc.uspr_zip_id);
```
Comment (cameroncaci) [inline migrations/app/schema/20240322144246_updateAddressesCounty.up.sql:10]: I get the feeling we're gonna want to increase the timeout or else CircleCI will exit it early. Here's a quick example of one I've done in the past:
```
-- Set temp timeout due to large record modification
-- Time is 5 minutes in milliseconds
SET statement_timeout = 300000;
SET lock_timeout = 300000;
SET idle_in_transaction_session_timeout = 300000;
```

**Review by cameroncaci — CHANGES_REQUESTED** [review]

File: migrations/app/schema/20240322144246_updateAddressesCounty.up.sql (hunk 1 above)
Comment (deandreJones) [reply migrations/app/schema/20240322144246_updateAddressesCounty.up.sql:10]: there is already a 60m timeout in the circle config- while running migrations
think we need more?

**Review by traskowskycaci — APPROVED** [review]
Migration ran successfully for me locally, county column has values. Nice!

File: migrations/app/schema/20240322144246_updateAddressesCounty.up.sql (hunk 1 above)
Comment (cameroncaci) [reply migrations/app/schema/20240322144246_updateAddressesCounty.up.sql:10]: Oh, I didn't know that we changed it to have a blanket 60m timeout. Remember back to when we were conducting some TGET migrations? Those were failing. If that 60m timeout is newer than those migrations (4 months ago) then this should be fine as is

Comment (deandreJones) [reply migrations/app/schema/20240322144246_updateAddressesCounty.up.sql:10]: been there for 5 years... so not that new

Ill add the above to add additional safety net

Comment (cameroncaci) [reply migrations/app/schema/20240322144246_updateAddressesCounty.up.sql:16]: Hm, it's likely not operating properly then. We've seen on a few occasions the migrations were timing out

**Review by cameroncaci — APPROVED** [review]

**Comment by cameroncaci** [comment]
Hm, getting some null values such as for 90210.
image

image

**Comment by deandreJones** [comment]
> Hm, getting some null values such as for 90210. image
> 
> image

refactored query, which got rid of the nulls.. that should have matched for me

**Comment by cameroncaci** [comment]
> > Hm, getting some null values such as for 90210. image
> > image
> 
> refactored query, which got rid of the nulls.. that should have matched for me


Hm, still getting some nulls
image

image

**Comment by cameroncaci** [comment]
@deandreJones 
I bet it's because of the manual SQL inserts under db dev populate not using the factories, inserting _after_ the migrations

**Comment by deandreJones** [comment]
> @deandreJones I bet it's because of the manual SQL inserts under db dev populate not using the factories, inserting _after_ the migrations. Hence probably why the makefile has a "TODO: deprecate" under that db dev populate

that makes sense, cuz when i run the script after the db is populated all is good

**Comment by cameroncaci** [comment]
I wonder where these inserts are coming from.. I see c/o Some Person showing up here: `scripts/pricing-acceptance`

**Review by cameroncaci — APPROVED** [review]
Works like a charm, the county null issue is separate

**Pull request merged**